contactform.php

```diff
@@ -41,29 +53,187 @@ public function __construct()
+            'ES' => 'https://addons.prestashop.com/es/429-seguridad-y-accesos?utm_source=back-office&utm_medium=native-contactform&utm_campaign=back-office-ES&utm_content=security',
+            'DE' => 'https://addons.prestashop.com/de/429-sicherheit-brechtigungen?utm_source=back-office&utm_medium=native-contactform&utm_campaign=back-office-DE&utm_content=security',
+            'IT' => 'https://addons.prestashop.com/it/429-security-access?utm_source=back-office&utm_medium=native-contactform&utm_campaign=back-office-IT&utm_content=security',
+            'NL' => 'https://addons.prestashop.com/nl/429-veiligheid-toegang?utm_source=back-office&utm_medium=native-contactform&utm_campaign=back-office-NL&utm_content=security',
+            'PL' => 'https://addons.prestashop.com/pl/429-bezpieczestwa-dostepu?utm_source=back-office&utm_medium=native-contactform&utm_campaign=back-office-PL&utm_content=security',
+            'PT' => 'https://addons.prestashop.com/pt/429-seguranca-acesso?utm_source=back-office&utm_medium=native-contactform&utm_campaign=back-office-PT&utm_content=security',
+            'RU' => 'https://addons.prestashop.com/ru/429-website-security-access?utm_source=back-office&utm_medium=native-contactform&utm_campaign=back-office-RU&utm_content=security',
+        );
+        $link = $urlsByLanguagesCodes['EN'];
+        $languageCode = strtoupper($this->context->language->language_code);
+
+        if (!empty($urlsByLanguagesCodes[$languageCode])) {
+            $link = $urlsByLanguagesCodes[$languageCode];
+        }
+        $linkText = $this->trans('PrestaShop Addons Marketplace', array(), 'Admin.Modules.Feature');
+
+        return sprintf('<a href="%1$s">%2$s</a>', $link, $linkText);
+    }
+
+    /**
+     * @return string
+     */
+    protected function renderForm()
+    {
+        $fieldsValue = array(
+            self::SEND_CONFIRMATION_EMAIL => Tools::getValue(
+                self::SEND_CONFIRMATION_EMAIL,
+                Configuration::get(self::SEND_CONFIRMATION_EMAIL)
+            ),
+            self::SEND_NOTIFICATION_EMAIL => Tools::getValue(
+                self::SEND_NOTIFICATION_EMAIL,
+                Configuration::get(self::SEND_NOTIFICATION_EMAIL)
+            )
+        );
+        $form = array(
+            'form' => array(
+                'legend' => array(
+                    'title' => $this->trans('Parameters', array(), 'Modules.Contactform.Admin'),
+                    'icon' => 'icon-envelope'
+                ),
+                'input' => array(
+                    array(
+                        'type' => 'switch',
+                        'label' => $this->trans('Send confirmation email to your customers', array(), 'Modules.Contactform.Admin'),
+                        'desc' => $this->trans('Click Yes and your customers will receive a confirmation email without the message content', array(), 'Modules.Contactform.Admin'),
```

Comment:
I think this wording could be improved to better guide the merchants:

> Choose Yes and your customers will receive a generic confirmation email including a tracking number after their message is sent. Note: to discourage spam, the content of their message won't be included in the email.

What do you think @LouiseBonnard @colinegin ?

Reply:
Ah oui !

Reply:
done